pepper-apis/config/application.conf.ctmpl

```diff
@@ -58,6 +58,8 @@
     "pubsub": {
         "enableHousekeepingTasks": {{$conf.Data.pubsub.enableHousekeepingTasks}},
         "housekeepingTasksSubscription": "{{$conf.Data.pubsub.housekeepingTasksSubscription}}",
+        "dsmTasksSubscription": "{{$conf.Data.pubsub.dsmTasksSubscription}}",
+        "dsmTasksResultTopic": {{$conf.Data.pubsub.dsmTasksResultTopic}}"
```

Comment:
Typo! Missing an opening double-quotes.

Reply:
dev server crashed because of bad config. I fixed it and redeployed to dev. Please pull the latest commit when you get a chance.

Reply:
Sorry, my fault! Thank you a lot!